Makefile

```diff
@@ -1,14 +1,14 @@
-TESTS = [features-kubernetes]
 # if IMAGE_TAG is not set, it will fall back to the version set in the CI
 # values file, then to the chart default.
-IMAGE_TAG =
+IMAGE_TAG ?= 
```

Comment:
I'm pretty sure it does - there are a few in lagoon makefile already?  being able to conditionally assign variables makes it much easier to override!

Reply:
The variables can still be overridden in the same way, can't they? I don't think there's a functional difference in this `Makefile`?

Reply:
Without the conditional assignment, the makefile can't be overridden from the command line (i.e to run test images from a different PR from core/remote, to avoid rebuilding)

Reply:
I don't think that's correct..
```
$ cat Makefile 
FOO = foo
test:
	@echo $(FOO)
```
```
$ make
foo
$ make FOO=bar
bar
```

Reply:
Ah, finally worked it out - it's the difference between overriding the variable in or outside the makefile
```
tobybellwood@pop-os:~/sites/test$ cat Makefile
FOO = foo
test:
	@echo $(FOO)
tobybellwood@pop-os:~/sites/test$ make
foo
tobybellwood@pop-os:~/sites/test$ make FOO=bar
bar
tobybellwood@pop-os:~/sites/test$ FOO=bar make
foo
tobybellwood@pop-os:~/sites/test$ vi Makefile
tobybellwood@pop-os:~/sites/test$ cat Makefile
FOO ?= foo
test:
	@echo $(FOO)
tobybellwood@pop-os:~/sites/test$ make
foo
tobybellwood@pop-os:~/sites/test$ make FOO=bar
bar
tobybellwood@pop-os:~/sites/test$ FOO=bar make
bar
```

Reply:
IMO explicit is better than implicit so would prefer to keep the requirement to specify values as make arguments, unless there's a compelling technical reason that setting these values via environment variables is required.

Reply:
fixed them up for explicit - meant to do it the other day, sorry!